### DISCUSSION TITLE
Media sync keeps trying to fetch something

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#1162] [Q&A] [answered]
For some reason I clicked on the "Update Media Folders" more than once, and each time M<sup>3</sup> behaved as if it was fetching media but it was all there. I checked repeatedly just to see what would happen and it kept "fetching" media. 
I don't recall this being so in the past. It normally just "blinks" and if there are no changes to the media then it wouldn't try download any?
(I have sped up the video x5 so that it is less boring.) As shown in the video, my internet speed this morning is about 90/90mb. 

https://user-images.githubusercontent.com/35364657/211902930-1fd2de85-f4ab-400c-a750-dbf7cefe8285.mp4

**Comment by mtdvlpr** [comment]
Hi @Chipperchap, it's definitely not normal! Could you look at the console and network tabs to see what he's trying to fetch? `ctrl` + `shift` + `I`.

**Reply by mtdvlpr** [reply]
The network tab needs to be open **before** you sync media.

**Comment by Chipperchap** [comment]
I don't know how to do the CTRL + SHIFT + I thing - but here goes:

OBS Success! Connected & authenticated.
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 01 Disinterested.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 02 Interested.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 03 Shy.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 04 Shy2.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 05 Youth.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 06 Youth2.png
instrument.js:108 [congMedia] [12-01-2023] 05-10 - Lesson 25 Why Did God Create the Earth - Exerpt.mp4
instrument.js:108 [congMedia] [12-01-2023] 06-11 - December 2022 Accounts.png
instrument.js:108 [congMedia] [13-01-2023] - 001.jpg
instrument.js:108 [congMedia] [13-01-2023] - 002 Successfully Coping with Peer Pressure-1.png
instrument.js:108 [congMedia] [13-01-2023] - 003 Successfully Coping with Peer Pressure-2.png
instrument.js:108 [congMedia] [13-01-2023] - 004 Successfully Coping with Peer Pressure-3.png
instrument.js:108 [congMedia] [13-01-2023] - 005 Successfully Coping with Peer Pressure-4.png
instrument.js:108 [congMedia] [13-01-2023] - 006.jpg
instrument.js:108 [congMedia] [13-01-2023] - 007.jpg
instrument.js:108 [congMedia] [13-01-2023] - 008.png
instrument.js:108 [congMedia] [13-01-2023] - 009.jpg
instrument.js:108 [congMedia] [13-01-2023] - 010.jpg
instrument.js:108 [jwOrg] [15-01-2023] 02-01 - Paragraph 7 - Collage - A sister praying at different times throughout the day - 1 -  With her young daughter before she goes to school - 2 -  During a.jpg
instrument.js:108 [jwOrg] [15-01-2023] 02-02 - Paragraph 16 - Collage - 1 -  A brother willingly forgives a brother who apologizes to him at the Kingdom Hall - 2 -  A beautiful forest of various colo.jpg
instrument.js:108 [jwOrg] [15-01-2023] 01-01 - Song 3 - Our Strength Our Hope Our Confidence.mp4
instrument.js:108 [jwOrg] [15-01-2023] 03-01 - Song 142 - Holding Fast to Our Hope.mp4
instrument.js:108 [jwOrg] [12-01-2023] 01-01 - Song 60 - It Means Their Life.mp4
instrument.js:108 [jwOrg] [12-01-2023] 03-01 - Return Visit - Prayer - 1Jo 5 - 14.mp4
instrument.js:108 [jwOrg] [12-01-2023] 04-01 - Song 143 - Keep Working Watching and Waiting.mp4
instrument.js:108 [jwOrg] [12-01-2023] 06-01 - Song 54 - This Is the Way.mp4
instrument.js:108 [jwOrg] [12-01-2023] 05-01 - A Bible student raising her hand at a congregation meeting of Jehovahs Witnesses.jpg
instrument.js:108 [jwOrg] [12-01-2023] 02-01 - A brother witnessing to a workmate during their lunch break.jpg
instrument.js:108 [perf] [getJwOrgMedia] 1126.9ms
instrument.js:108 [perf] [createMediaNames] 8.6ms
instrument.js:108 [perf] [getCongMedia] 4.1ms
instrument.js:108 [perf] [syncJWorgMedia] 408.9ms
instrument.js:108 [perf] [syncCongMedia] 1696.6ms
instrument.js:108 [perf] [total] 2854.9ms
instrument.js:108 [perf] [jworg Fetch] from cache: 282.9MB
instrument.js:108 [perf] [cong Fetch] from cache: 22.4MB
instrument.js:108 [perf] [cong Fetch] from live: 4.3MB
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 01 Disinterested.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 02 Interested.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 03 Shy.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 04 Shy2.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 05 Youth.png
instrument.js:108 [congMedia] [12-01-2023] 04-10 - 06 Youth2.png
instrument.js:108 [congMedia] [12-01-2023] 05-10 - Lesson 25 Why Did God Create the Earth - Exerpt.mp4
instrument.js:108 [congMedia] [12-01-2023] 06-11 - December 2022 Accounts.png
instrument.js:108 [congMedia] [13-01-2023] - 001.jpg
instrument.js:108 [congMedia] [13-01-2023] - 002 Successfully Coping with Peer Pressure-1.png
instrument.js:108 [congMedia] [13-01-2023] - 003 Successfully Coping with Peer Pressure-2.png
instrument.js:108 [congMedia] [13-01-2023] - 004 Successfully Coping with Peer Pressure-3.png
instrument.js:108 [congMedia] [13-01-2023] - 005 Successfully Coping with Peer Pressure-4.png
instrument.js:108 [congMedia] [13-01-2023] - 006.jpg
instrument.js:108 [congMedia] [13-01-2023] - 007.jpg
instrument.js:108 [congMedia] [13-01-2023] - 008.png
instrument.js:108 [congMedia] [13-01-2023] - 009.jpg
instrument.js:108 [congMedia] [13-01-2023] - 010.jpg
instrument.js:108 [jwOrg] [15-01-2023] 02-01 - Paragraph 7 - Collage - A sister praying at different times throughout the day - 1 -  With her young daughter before she goes to school - 2 -  During a.jpg
instrument.js:108 [jwOrg] [15-01-2023] 02-02 - Paragraph 16 - Collage - 1 -  A brother willingly forgives a brother who apologizes to him at the Kingdom Hall - 2 -  A beautiful forest of various colo.jpg
instrument.js:108 [jwOrg] [15-01-2023] 01-01 - Song 3 - Our Strength Our Hope Our Confidence.mp4
instrument.js:108 [jwOrg] [15-01-2023] 03-01 - Song 142 - Holding Fast to Our Hope.mp4
instrument.js:108 [jwOrg] [12-01-2023] 01-01 - Song 60 - It Means Their Life.mp4
instrument.js:108 [jwOrg] [12-01-2023] 03-01 - Return Visit - Prayer - 1Jo 5 - 14.mp4
instrument.js:108 [jwOrg] [12-01-2023] 04-01 - Song 143 - Keep Working Watching and Waiting.mp4
instrument.js:108 [jwOrg] [12-01-2023] 06-01 - Song 54 - This Is the Way.mp4
instrument.js:108 [jwOrg] [12-01-2023] 05-01 - A Bible student raising her hand at a congregation meeting of Jehovahs Witnesses.jpg
instrument.js:108 [jwOrg] [12-01-2023] 02-01 - A brother witnessing to a workmate during their lunch break.jpg
instrument.js:108 [perf] [getJwOrgMedia] 590.1ms
instrument.js:108 [perf] [createMediaNames] 0.6ms
instrument.js:108 [perf] [getCongMedia] 3.6ms
instrument.js:108 [perf] [syncJWorgMedia] 423.0ms
instrument.js:108 [perf] [syncCongMedia] 2676.3ms
instrument.js:108 [perf] [total] 3292.9ms
instrument.js:108 [perf] [jworg Fetch] from cache: 282.9MB
instrument.js:108 [perf] [cong Fetch] from cache: 22.4MB
instrument.js:108 [perf] [cong Fetch] from live: 4.3MB

<html>
<body>
<ol class="tree-outline hide-selection-when-blurred issues" role="tree" tabindex="-1" style="box-sizing: border-box; min-width: 0px; min-height: 0px; padding: 0px; margin: 0px; z-index: 0; position: relative; overflow: auto; --issue-indent:8px; list-style-type: none; color: rgb(32, 33, 36); font-family: &quot;Segoe UI&quot;, Tahoma, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li role="treeitem" class="issue parent" aria-expanded="false" style="box-sizing: border-box; min-width: 0px; min-height: 16px; padding-top: 0px; padding-right: 8px; padding-bottom: 0px; padding-left: var(--issue-indent); overflow: hidden; flex: 0 0 auto; transition: background-color 0.2s ease 0s; border-top-color: ; border-top-style: ; border-right-color: ; border-right-style: ; border-bottom-color: ; border-bottom-style: ; border-left-color: ; border-left-style: ; border-image-source: ; border-image-slice: ; border-image-width: ; border-image-outset: ; border-image-repeat: ; border-width: 0px 0px 1px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center;"><div class="header" style="box-sizing: border-box; min-width: 0px; min-height: 0px; display: flex; flex-direction: row; align-items: center; padding: 6px 0px; cursor: pointer; width: 510.266px;"><div class="title" style="box-sizing: border-box; min-width: 0px; min-height: 0px; flex: 1 1 0%; font-size: 14px; color: var(--color-text-primary); font-weight: 400; user-select: text;">Audit usage of navigator.userAgent, navigator.appVersion, and navigator.platform</div></div></li><li role="treeitem" class="issue parent expanded" aria-expanded="true" style="box-sizing: border-box; min-width: 0px; min-height: 16px; padding-top: 0px; padding-right: 8px; padding-bottom: 0px; padding-left: var(--issue-indent); overflow: hidden; flex: 0 0 auto; transition: background-color 0.2s ease 0s; border-top-color: ; border-top-style: ; border-right-color: ; border-right-style: ; border-bottom-color: ; border-bottom-style: ; border-left-color: ; border-left-style: ; border-image-source: ; border-image-slice: ; border-image-width: ; border-image-outset: ; border-image-repeat: ; border-width: 0px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center; background: var(--color-background);"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span><div class="header" style="box-sizing: border-box; min-width: 0px; min-height: 0px; display: flex; flex-direction: row; align-items: center; padding: 6px 0px; cursor: pointer; width: 510.266px;"><devtools-icon class="leading-issue-icon" title="A page error issue: the page is not working correctly" style="display: inline-block; white-space: nowrap; box-sizing: border-box; min-width: 0px; min-height: 0px; margin: 0px 7px;"><div class="icon-basic" style="width: 16px; height: 16px; display: block; -webkit-mask-image: url(&quot;devtools://devtools/bundled/Images/issue-cross-icon.svg&quot;); -webkit-mask-position: center center; -webkit-mask-repeat: no-repeat; -webkit-mask-size: 99%; background-color: var(--icon-color, var(--issue-color-red));"></div></devtools-icon><devtools-adorner class="aggregated-issues-count" aria-label="countWrapper" style="display: inline-flex; box-sizing: border-box; min-width: 0px; min-height: 0px; padding-right: 7px;"><slot name="content" style="display: inline-flex; box-sizing: border-box; height: 14px; line-height: 13px; padding: 0px 6px; font-size: 8.5px; color: var(--override-adorner-text-color,var(--color-text-primary)); background-color: var(--override-adorner-background-color,var(--color-background-elevation-1)); border: 1px solid var(--override-adorner-border-color,var(--color-details-hairline)); border-radius: 10px;"><span slot="content" style="height: 10px; box-sizing: border-box; min-width: 0px; min-height: 0px;">11</span></slot></devtools-adorner><div class="title" style="box-sizing: border-box; min-width: 0px; min-height: 0px; flex: 1 1 0%; font-size: 14px; color: var(--color-text-primary); font-weight: 450; user-select: text;">Indicate whether a cookie is intended to be set in a cross-site context by specifying its SameSite attribute</div></div></li><ol class="children body issue-kind-page-error expanded" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; list-style-type: none; padding-left: calc(var(--issue-indent) + 43px); border-bottom: 1px solid var(--color-details-hairline-light); padding-top: 6px; position: relative; padding-bottom: 26px; padding-right: 8px; display: block;"><li role="treeitem" class="always-parent" style="box-sizing: border-box; min-width: 0px; min-height: 16px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center;"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span><devtools-markdown-view style="box-sizing: border-box; min-width: 0px; min-height: 0px;"><div class="message" style="line-height: 20px; font-size: 14px; color: var(--color-text-secondary); margin-bottom: 4px; user-select: text;"><p style="margin-bottom: 16px; margin-block-start: 2px;">Because a cookie’s<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">SameSite</code><span> </span>attribute was not set or is invalid, it defaults to<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">SameSite=Lax</code>, which prevents the cookie from being set in a cross-site context. This behavior protects user data from accidentally leaking to third parties and cross-site request forgery.</p><p style="margin-bottom: 16px; margin-block-start: 2px;">Resolve this issue by updating the attributes of the cookie:</p><ul style="list-style-type: none; list-style-position: inside; padding-inline-start: 0px;"><li style="margin-top: 8px; display: list-item;">Specify<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">SameSite=None</code><span> </span>and<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">Secure</code><span> </span>if the cookie is intended to be set in cross-site contexts. Note that only cookies sent over HTTPS may use the<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">Secure</code><span> </span>attribute.</li><li style="margin-top: 8px; display: list-item;">Specify<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">SameSite=Strict</code><span> </span>or<span> </span><code style="color: var(--color-text-primary); font-size: 12px; user-select: text; cursor: text; background: var(--color-background-elevation-1);">SameSite=Lax</code><span> </span>if the cookie should not be set by cross-site requests.</li></ul></div></devtools-markdown-view></li><li role="treeitem" class="always-parent parent expanded affected-resources-label" aria-expanded="true" style="box-sizing: border-box; min-width: 0px; min-height: 16px; margin-top: 5px; font-size: 10px; font-weight: 500; letter-spacing: 0.06em; text-transform: uppercase; color: var(--color-text-primary); display: flex; text-overflow: ellipsis; white-space: normal; position: relative; align-items: center;">AFFECTED RESOURCES</li><ol class="children expanded affected-resources" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; list-style-type: none; padding: 3px 0px 0px; position: relative; user-select: text; display: block;"><li role="treeitem" class="parent expanded" aria-expanded="true" style="box-sizing: border-box; min-width: 0px; min-height: 16px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center; margin-top: 0px; padding: 2px 5px 0px; background: var(--color-background-elevation-0);"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span><div class="affected-resource-label" style="box-sizing: border-box; min-width: 0px; min-height: 0px; font-size: 14px; line-height: 20px; color: var(--color-text-primary); position: relative; cursor: pointer;">1 cookie</div></li><ol class="children expanded" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; list-style-type: none; padding: 6px 0px 9px 5px; display: block; background: var(--color-background-elevation-0); margin-bottom: 10px;"><li role="treeitem" style="box-sizing: border-box; min-width: 0px; min-height: 16px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center;"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span>

Name | Domain & Path
-- | --
day1host | .4shared.com/

</li></ol><li role="treeitem" class="parent selected" aria-expanded="false" tabindex="0" aria-selected="true" style="box-sizing: border-box; min-width: 0px; min-height: 16px; outline-width: 0px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: center; margin-top: 0px; padding: 2px 5px 0px;"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span><div class="affected-resource-label" style="box-sizing: border-box; min-width: 0px; min-height: 0px; font-size: 14px; line-height: 20px; color: var(--color-text-primary); position: relative; cursor: pointer;">10 requests</div></li></ol></ol></ol>
</body>
</html>

I hope you can make head or tail of all that!

**Reply by mtdvlpr** [reply]
@Chipperchap, it seems the issue is the 4.3MB that keeps getting fetched from your cong server, instead of from the cache.

**Reply by mtdvlpr** [reply]
I will test with my own 4shared server. If I look at our Sentry logs, it seems fetching the directory contents of 4shared is broken again.

**Reply by mtdvlpr** [reply]
If you open those meeting days to manage the media. Does it list some files as from the cong server but not downloaded yet?

**Comment by Chipperchap** [comment, marked as answer]
Hi @mtdvlpr 
Sorry for the delay - our time zones are opposite each other. 
I went into 4Shared on the web and deleted all folders.
I then started M<sup>3</sup> and updated and all is fixed. 
I logged into the K-Hall to double check there and it is running properly as well. 
So it must have been, as you suggested, a corrupted folder somehow. 
Thank you so much for your assistance! Really appreciate it. 😀
